src/partials/hero.html

```diff
@@ -1,3 +1,27 @@
 <section class="hero">
-
-</section>
\ No newline at end of file
+  <div class="hero-container">
```

Comment:
After tag `section ` should be <div class="container">. In container should be added all your code.

Reply:
Done